lib/std/debug/SelfInfo/Elf.zig

```diff
@@ -114,6 +114,10 @@ pub const can_unwind: bool = s: {
             .x86,
             .x86_64,
         },
+        .serenity => &.{
+            .x86_64,
+            .aarch64,
+        },
```

Comment:
Do you feel like adding riscv64 while you're at it?

Reply:
Done; though I've not tested that yet.